### DISCUSSION TITLE
P7 input with 8.2.4 PRORES: DV to SDR (v2.0.5)

### DISCUSSION THREAD

**Discussion by brandonlacombe** [R3S3t9999/DoVi_Scripts#25] [General]
This maybe a bug.

If you input a P7 dolby vision file to the DV to SDR option it will do an x265 encode with avisynth baking the fel.

I expected it to do a prores encode with cm_offline instead.

If I wanted to output the 100-nit from a P7 dovi file is this supposed to be the correct way of doing it?
I may be missing a step.
I am confused as to whether baking the fel prior to using cm_offline is necessary for the correct output when starting from a P7 file.

**Comment by R3S3t9999** [comment]
Thanks for reporting. Please try the [latest beta](https://drive.google.com/file/d/128gq8aDUTKA_aT7SQsM9dkjA1EP1sosR/view?usp=drive_link).
